test/images/resource-consumer/BASEIMAGE

```diff
@@ -1,4 +1,4 @@
-amd64=gcr.io/google-containers/debian-base-amd64:0.1
-arm=gcr.io/google-containers/debian-base-arm:0.1
-arm64=gcr.io/google-containers/debian-base-arm64:0.1
-ppc64le=gcr.io/google-containers/debian-base-ppc64le:0.1
+amd64=gcr.io/google-containers/debian-base-amd64:0.2
```

Comment:
You may need to bump the version for resource-consumer image in `test/images/resource-consumer/VERSION`